File: Project/Source/Cells/_TorrentCell.m
//
//  TorrentCell.m
//  iTransmission
//
//  Created by Mike Chen on 10/3/10.
//  Copyright (c) 2010 __MyCompanyName__. All rights reserved.
//

#import <iTransmission-Swift.h>
#import "_TorrentCell.h"
#import "PDColoredProgressView.h"

@implementation _TorrentCell

@synthesize nameLabel;
@synthesize upperDetailLabel;
@synthesize lowerDetailLabel;
@synthesize progressView;
@synthesize controlButton;


- (void)setProgress:(float)progress
{
    [self.progressView setProgress:progress];
    [self setNeedsDisplay];
}

- (IBAction)pausedPressed:(id)sender
{

}

@end

File: Project/Source/Cells/TorrentCell.swift
//
//  TorrentCell.swift
//  iTransmission
//
//  Created by Vitalii Yevtushenko on 19.07.2019.
//

import UIKit

@objc
class TorrentCell: UITableViewCell {

    @objc static let identifier: String = "TorrentCellIdentifier"

    @IBOutlet var nameLabel: UILabel!
    @IBOutlet var progressView: UIProgressView!
    @IBOutlet var upperDetailLabel: UILabel!
    @IBOutlet var lowerDetailLabel: UILabel!
    @IBOutlet var controlButton: ControlButton!

    @objc func useGreenColor() {
        progressView.tintColor = UIColor(named: "progress-green")
    }

    @objc func useBlueColor() {
        progressView.tintColor = UIColor(named: "progress-blue")
    }
}
